LEGO1/lego/legoomni/include/act3.h

```diff
@@ -5,6 +5,7 @@
 #include "legogamestate.h"
 #include "legostate.h"
 #include "legoworld.h"
+#include "mxgeometry/mxmatrix.h"
```

Comment:
I don't think we need this include? At most a forward declaration of `class Matrix4` should do I think